pkg/exchange/max/exchange.go

```diff
@@ -311,6 +315,47 @@ func (e *Exchange) queryClosedOrdersByLastOrderID(ctx context.Context, symbol st
 	return types.SortOrdersAscending(orders), nil
 }
 
+func (e *Exchange) queryClosedOrdersByTime(ctx context.Context, symbol string, since, until time.Time) (orders []types.Order, err error) {
+	if err := e.closedOrderQueryLimiter.Wait(ctx); err != nil {
+		return orders, err
+	}
+
+	market := toLocalSymbol(symbol)
+	walletType := maxapi.WalletTypeSpot
+	if e.MarginSettings.IsMargin {
+		walletType = maxapi.WalletTypeMargin
+	}
+
+	req := e.v3client.NewGetWalletClosedOrdersRequest(walletType).Market(market).Limit(1000)
+
+	if !until.IsZero() {
+		req.Timestamp(until)
+	}
+
+	maxOrders, err := req.Do(ctx)
```

Comment:
and, what's the default ordering?

Reply:
desc. there are four order type - asc, desc, asc_updated_at, desc_updated_at

Reply:
but we should use asc here

Reply:
ok!